bgpd/bgp_updgrp.c

```diff
@@ -49,6 +49,7 @@
 #include "bgpd/bgp_debug.h"
 #include "bgpd/bgp_errors.h"
 #include "bgpd/bgp_fsm.h"
+#include "bgpd/bgp_addpath.h"
```

Comment:
why do you need to include this header file? I can see the declaration being added to bgp_updgrp.h.

Reply:
Thanks, missed removing this because initially, I created it in bgp_addpath.c.

Reply:
`bgp_addpath.h` is needed for `bgp_addpath_tx_path()`.

Reply:
Got it, sorry.